supabase/migrations/20260715175642_repair_rbac_function_grants.sql

```
@@ -0,0 +1,82 @@
+-- The squashed baseline retained a dead pre-RBAC trigger function. The current
+-- schema has no trigger or other dependency on it, and its body references
+-- columns and types that no longer exist.
+DROP FUNCTION IF EXISTS public.generate_org_user_on_org_create();
+-- Keep the API-key listing RPC signed-in only. Reassert explicit grants so the
+-- function cannot regain anonymous execution through default ACLs.
+REVOKE ALL ON FUNCTION public.get_org_apikeys(uuid) FROM PUBLIC;
+REVOKE ALL ON FUNCTION public.get_org_apikeys(uuid) FROM anon;
+REVOKE ALL ON FUNCTION public.get_org_apikeys(uuid) FROM authenticated;
+GRANT EXECUTE ON FUNCTION public.get_org_apikeys(uuid) TO authenticated, service_role;
```

SQLFluff (4.2.2)

[error] Line is too long (86 > 80).

(LT05)